site2/docs/concepts-messaging.md

```diff
@@ -411,4 +411,32 @@ Message deduplication makes Pulsar an ideal messaging system to be used in conju
 
 > More in-depth information can be found in [this post](https://streaml.io/blog/pulsar-effectively-once/) on the [Streamlio blog](https://streaml.io/blog)
 
+## Delayed Message Delivery
+Delayed Message Delivery enables you to consume new message later, instead of immediately as normal. In this mechanism, messages is stored in memory after publish to broker, after specific delayed time passed, the message will be delivered to consumer.  
```

Comment:
```suggestion
Delayed message delivery enables you to consume a message later rather than immediately. In this mechanism, a message is stored in memory after published to a broker, and it is delivered to a consumer once the specific delayed time is passed. 
```

Reply:
@Anonymitaet  done